device/cuda/src/seeding/seed_finding.cu

```diff
@@ -180,7 +180,8 @@ vecmem::data::vector_buffer<seed> seed_finding::operator()(
     // counting kernel for.
     const unsigned int nDoubletCountThreads = WARP_SIZE * 2;
     const unsigned int nDoubletCountBlocks =
-        sp_grid_prefix_sum_buff.size() / nDoubletCountThreads + 1;
+        (sp_grid_prefix_sum_buff.size() + nDoubletCountThreads - 1) /
+        nDoubletCountThreads;
```

Comment:
I am also happy with these changes in the CUDA code. But again, please add 1-2 sentences about these into the PR's description.